src/main/features/claim/routes/fees.ts

```diff
@@ -16,6 +16,7 @@ export default express.Router()
   .get(Paths.feesPage.uri, async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
     const draft: Draft<DraftClaim> = res.locals.claimDraft
     const claimAmount: number = await claimAmountWithInterest(draft.document)
+
```

Comment:
I think it would be good to revert that change as it does not add anything.

Reply:
done